### ISSUE TITLE
I already have a good experience of ubuntu touch. It seems that my nexus 5 has no good partition size. Maybe you could help me to solve this.

### ISSUE THREAD

**Issue by lool-lauris** [ubports/ubports-installer#1510]
**UBports Installer `0.6.0-beta` (snap)**
Environment: `Ubuntu Core 20 linux 5.4.0-52-generic x64 NodeJS v12.16.3`
Device: hammerhead
Target OS: Ubuntu Touch
Settings: `undefined`
OPEN-CUTS run: https://ubports.open-cuts.org/run/5fa81a4b9702a600079db789
Log: *N/A*

Probably by resizing the partitions of this nexus 5

**Comment by NeoTheThird** [comment]
Hi, sorry, can you clarify? Did the installation succeed or not? Where did you run into an issue with your partitions?

**Comment by lool-lauris** [comment]
Hi,

I didn't succeed.
Actually, I have a problem with this phone. I tried to install different rom on it in order to get it back. Not so long ago, I couldn't even install UT, the installation stopped in the process (during the "Sending" phase of the installer). The message said "out of space" or something similar.

I suspected a partitioning problem. I followed this method ( https:/forum.xda-developers.comgoogle-nexus-5generalguide-repartition-nexus5-to-increase-t3509880 ) to redo the partitions.
From then on, the installation process went all the way to the end. I thought I had finally managed to save this phone.
But at the end of the installation, the phone reboots and display "Installing update" for several minutes, then at the end of this operation, the "google" home screen appears and the phone shuts down.

This morning, I redo the installation. and there for the first time is displayed at the end the ubports home screen but remains stuck to this display.

Here is the message obtained after having clicked on the "FAIL" button of the "Report your result" window and passing on the form :
Error: unhandled rejection at [object Promise]: TypeError: Cannot read property 'title' of null

For info, I use UBports Installer (0.6.0-beta) - as a snap. I tried different options: "Bootstrap" checked alone, then "Bootstrap" and "Format system partitions" checked, then "Bootstrap" and "Format system partitions" and "Wipe Userdata" checked.

It would be very nice to help me to save that phone. Thank you for your help.
